ee/se/modules/380-metallb/enabled

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Copyright 2021 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+source /deckhouse/shell_lib.sh
+
+function __main__() {
+  enabled::disable_module_if_cluster_is_not_bootstraped
+
+  echo "true" > "$MODULE_ENABLED_RESULT"
+}
+
+enabled::run "$@"
```

Comment:
enabled-scripts are depricated. We should use module.yml

Reply:
File deleted